examples/python/gRPC/tf/gRPC_fb_getTf.py

```diff
@@ -13,9 +13,13 @@
 
 builder = flatbuffers.Builder(1024)
 channel = get_gRPC_channel("local")
-PROJECT_NAME = "plantmap01"
+PROJECT_NAME = "testproject"
 projectUuid = getProject(builder, channel, PROJECT_NAME)
 
+if not projectUuid:
+    print("project not found")
+    exit
```

Comment:
Aren't there brackets missing? 
```suggestion
    exit()
```